gensrc/thrift/PlanNodes.thrift

```diff
@@ -855,6 +855,7 @@ enum TopNAlgorithm {
    3: optional bool has_global_limit
    4: optional TopNAlgorithm top_n_algorithm
    5: optional i64 partition_inner_limit
+   6: optional bool is_two_phase_global
```

Comment:
maybe we could add an enum phase for better extension

Reply:
done